assets/styles/styles.css

```diff
@@ -903,3 +903,6 @@ footer p a {
     height: 15em;
   }
 }
+#calculate-sleep {
+  padding-bottom: 2rem;
+}
```

Comment:
Change requested as per comment bellow :)

Reply:
Changes made and pushed branch for review.